services/virtual-assistant/src/virtual_assistant/assistant/watson.py

```diff
@@ -107,6 +102,23 @@ def get_feedback_command_params(
     return [summary, description, labels]
 
 
+def get_service_account_command_params(watson_msg: str) -> Tuple[str, str, str]:
+    """
+    Extracts the params from the message of the service account command.
+
+    Example feedback command and params:
+    /create_service_account
+    <|name|>test1<|name|>
+    <|description|>Now, provide a short description for your service account.<|description|>
+    <|environment|>stage<|environment|>
```

Comment:
You might want to update this docstring. Threw me off for a sec! 😄

Reply:
Whoops, done